test/Libraries/RevitIntegrationTests/SelectionTests.cs

```diff
@@ -896,6 +904,10 @@ private void AssertElementCategoryOnCollection(List<object> collection, string e
                 var elem = elements[i] as Revit.Elements.Element;
                 Assert.AreEqual(expectedCategoryString, elem.GetCategory.Name);
             }
+
+            var selectionFilter = selectNode.Filter as CategoryElementSelectionFilter<Element>;
+            Assert.IsTrue(selectionFilter.Category == BuiltInCategory.OST_Walls);
+
```

Comment:
I tested it today locally, ran the system test, but it failed as there is an assert failed. 
failed at `Assert.NotNull(elements);`, I will check it again.

Reply:
> @wangyangshi can you confirm that the new and modified tests are running correctly?


I merged it as want to integrate this fix to Revit2024 although the test failed.
![image](https://user-images.githubusercontent.com/62596335/207368206-2f9c9518-e717-41de-b6bc-384b8e8e26b8.png)
from debug, the elements is null, this is unexpected compared with other tests, like `SelectModelElementsByCategory`
![image](https://user-images.githubusercontent.com/62596335/207368864-38ca7a94-f528-4664-a051-8b44ccfaccda.png)
@saintentropy any ideas about this?